musica-fortran/src/micm/MICM_mod.F90

```diff
@@ -0,0 +1,69 @@
+module libmicm
```

Comment:
```suggestion
module micm
```

please rename to micm. Our fortran style is to call the module the component name and any types <module_name>_t

Reply:
Fixed